tests/integration/api/test_gcp_integration.py

```diff
@@ -0,0 +1,64 @@
+import os
+from datadog import api as dog
+from datadog import initialize
+
+API_KEY = os.environ.get("DD_TEST_CLIENT_API_KEY", "a" * 32)
+APP_KEY = os.environ.get("DD_TEST_CLIENT_APP_KEY", "a" * 40)
+API_HOST = os.environ.get("DATADOG_HOST")
+
+
+class TestGcpIntegration:
+
+    test_project_id = "datadog-apitest"
+    test_client_email = "api-dev@datadog-sandbox.iam.gserviceaccount.com"
+
+    @classmethod
+    def setup_class(cls):
+        initialize(api_key=API_KEY, app_key=APP_KEY, api_host=API_HOST)
+
+    def test_gcp_create(self):
+        output = dog.Gcp.create(
+            type="service_account",
+            project_id=self.test_project_id,
+            private_key_id="123456789abcdefghi123456789abcdefghijklm",
```

Comment:
is this a sensitive key, If not could we make it more obvious it is not?

Reply:
nope that's just numbers and letters in order twice. i.e. 123456789 twice, and abcdefghi twice.  I can change it to something more obvious.

Reply:
I've changed this to `fake_private_key_id` for clarity.